solidity/contracts/Zoltar.sol

```diff
@@ -196,11 +194,12 @@ contract Zoltar {
 			ReputationToken(address(universe.reputationToken)).burn(migrator, amount);
 		}
 
-		uint192 childUniverseId = uint192((universeId << 2) + outcome + 1);
-		Universe memory childUniverse = universes[childUniverseId];
-		ReputationToken(address(childUniverse.reputationToken)).mint(recipient, amount);
+		for (uint8 i = 1; i < Constants.NUM_OUTCOMES + 1; i++) {
+			if (singleOutcome < 3 && i != singleOutcome + 1) continue;
```

Comment:
See associated comment above.
```suggestion
			if (singleOutcome != type(uint8).max && i != singleOutcome + 1) continue;
```

Note: This fails in favor of minting nothing if there is somehow a bug where someone passes a singleOutcome other than the sentinal value or a valid outcome.